scripts/create_latest_tag_step.sh

```diff
@@ -2,8 +2,8 @@
 
 GIT_URL=$1
 
-git config --global user.email "builds@travis-ci.com"
-git config --global user.name "Travis CI"
+git config --global user.email "prow@kyma.cx"
+git config --global user.name "Prow CI"
 export GIT_TAG=latest
-git tag $GIT_TAG -f -a -m "Generated tag from TravisCI"
+git tag $GIT_TAG -f -a -m "Generated tag from ProwCI"
```

Comment:
You gonna hate me, but consider renaming file `convert.sh`. It looks that this script not only converts bundles to targz, but rather prepare some artifacts to be released.

Reply:
I will rename it to `prepare-release-assets`.